go/dpagg/mean.go

```diff
@@ -213,6 +215,101 @@ func (bm *BoundedMeanFloat64) Result() float64 {
 	return clamped
 }
 
+// ComputeConfidenceInterval computes a confidence interval that contains the true mean with
+// probability greater than or equal to 1 - alpha. The computation is based exclusively on the
```

Comment:
should we say maybe that count and sum based only on the noised value and privacy params?
Because from this sentence (and if reader doesn't know details) it is not clear why this leads to not using extra budget.

Reply:
"The computation is based exclusively on the confidence intervals for sum and count, which are based only on the noised value and privacy params" is this ok?

Reply:
parameters.